CommonMark/__init__.py

```diff
@@ -1,5 +1,5 @@
 # flake8: noqa
-from __future__ import unicode_literals, absolute_import
```

Comment:
these import are used to maintain compatibility with python 2, which this library still supports, and will for a while: https://github.com/rtfd/CommonMark-py/blob/master/.travis.yml#L4